ci-operator/config/openshift/release/openshift-release-master__nightly-4.15.yaml

```diff
@@ -608,18 +607,32 @@ tests:
     cluster_profile: gcp
     workflow: openshift-e2e-gcp-xpn
   timeout: 6h0m0s
-- as: e2e-aws-ovn-upgrade-rollback-oldest-supported
+- as: aws-ovn-upgrade-rollback-oldest-supported
   interval: 48h
   steps:
     cluster_profile: aws-2
     dependencies:
       OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:oldest-supported-z-rollback
       OPENSHIFT_UPGRADE_RELEASE_IMAGE_OVERRIDE: release:latest
     env:
-      BASE_DOMAIN: aws-2.ci.openshift.org
-      TEST_TYPE: upgrade-conformance
-      TEST_UPGRADE_OPTIONS: abort-at=100
-    workflow: openshift-upgrade-aws
+      BASE_DOMAIN: qe.devcluster.openshift.com
```

Comment:
I think we also need to change `cluster_profile` to a QE one to be able to use QE domain?

Reply:
Right. Thank you for pointing it out.